### ISSUE TITLE
autocomplete for symbol in cljs file adds require from clj file

### ISSUE THREAD

**Issue by zubairAmperity** [BetterThanTomorrow/calva#2308]
The autocomplete doesn't include an symbol defined in a fn, but does include an example of an alias of a namespace not included in the project. 

That namespace comes from a .clj file, and i'm working in a .cljs file, so it shouldn't even be an option for autocomplete

![image](https://github.com/BetterThanTomorrow/calva/assets/99685017/1c561651-abd6-4065-80b3-2075c3741b88)

**Comment by PEZ** [comment]
Hi! That looks wrong. If there isn't an issue about this on clojure-lsp, I think @ericdallo would want one. If you can create a minimal reproduction, that would be extra awesome. 🙏

**Comment by ericdallo** [comment]
yes, clojure-lsp is aware of that during complete and that shouldn't happen, so we need a repro, feel free to create a issue when having one
